src/utils.rs

```diff
@@ -2,20 +2,10 @@
 //!
 //! This module is _not_ exposed to the public directly.
 
-use error::Result;
-
 /// A retrieved offset for a particular partition in the context of an
 /// already known topic.
 #[derive(Debug)]
 pub struct PartitionOffset {
-    pub offset: Result<i64>,
-    pub partition: i32,
-}
```

Comment:
this used to be what now is replaced by `ProduceTopicPartitionConfirm`: as suspected the size of the struct got smaller!

```rust

#[test]
fn test_size() {
    use std::mem;
    println!("new version: {}", mem::size_of::<ProduceTopicPartitionConfirm>());
    println!("old version: {}", mem::size_of::<PartitionOffset>());
}
```

```
new version: 24
old version: 56
```

:cool: